ONBOARDING.md

```diff
@@ -52,6 +52,7 @@ To run on Android, from the `client/ionic` directory run:
 ```
 ionic build
 ionic capacitor copy android
+npx cap update android
```

Comment:
Doesn't `ionic capacitor copy android` do this?
If not please use `ionic capacitor update android`. Npx is not a dep off this project and might not be globally installed.

Reply:
npx will be installed by default with npm > 5.2.0 (so as long as people use Node 10, it should be there). That being said, it is clearer to use the ionic CLI instead.

Reply:
> Doesn't `ionic capacitor copy android` do this?

It didn't for me.

> If not please use `ionic capacitor update android`.

I can send a new PR for consistency.